adding_species_manuscript.tex

```diff
@@ -852,159 +890,86 @@ \subsection*{\texorpdfstring{\emph{Bos taurus} (cattle)}{Bos taurus (cattle)}}
+might have slightly higher linkage disequilibrium than observed in real cattle genomes.
 However, we anticipate that this would affect patterns less
 than selection due to domestication and selective breeding,
-which are not modeled here.
+which are as of yet not modeled at all in the simulations of \stdpopsim.
 
 \section*{Conclusion}
     \label{conclusion}
 
 As our ability to sequence genomes continues to advance, the need for
-population genomic simulation of new model and non-model organism genomes is
-becoming acute. So too is the concomitant need for an expandable framework
-for implementing such simulations for species of interest and
-the resources for understanding when and how to do so.
-
-Simulating species of interest, both model and non-model, presents significant challenges
+population genomic simulation of new model and non-model organisms is becoming acute.
+So too is the concomitant need for an expandable framework for implementing such simulations and guidance for how to do so.
+Generating realistic whole-genome simulations presents significant challenges
 in coding and the choice of parameter values on which to base the simulation.
-\stdpopsim is a resource that is uniquely poised to address these 
-challenges as it provides easy access to simulations incorporating 
-species-specific information, easy inclusion of new species genomes,
-and the choices of new species to include are driven by the
+\Stdpopsim is a resource that is uniquely poised to address these 
+challenges as it provides easy access to state-of-the-art simulation engines and practices and an easy procedure for including new species genomes.
+Moreover, the choices of new species to include are driven by the
 needs of the population genomics community. In this manuscript we 
-describe the expansion of \stdpopsim in two ways: the expansion
-of its underlying framework to incorporate new evolutionary processes
-such as gene conversion, which broadens the diversity of species that can
+describe the expansion of \stdpopsim in two ways:
+the addition of new features to the simulation framework that incorporate new evolutionary processes,
+such as non-crossover recombination, which broaden the diversity of species that can
 be realistically modeled; and the considerable expansion of the catalog itself
 to include more species and demographic models. 
 
-We also present basic considerations for implementing
-population genomic simulations, agnostic to simulation software, based on
-insights from the community-driven process of expanding the \stdpopsim catalog. 
-We describe the steps of determining if a species-specific population
-genomic simulation is appropriate for the species and question, what
-data is necessary and why, special considerations for finding and using
-that data, how to proceed when some of that data is not available,
-and why we encourage everyone implementing simulations to have their
+We also compiled a series of guidelines for implementing
```

Comment:
"compiled" -> "developed" or "formulated"? (Compiled implies they already existed in bits and pieces elsewhere.)